packages/headless/src/app/engine.browser.test.ts

```diff
@@ -25,6 +25,8 @@ describe('engine', () => {
       },
       reducers: {},
     };
+    document.cookie = '';
+    localStorage.clear();
```

Comment:
Now that we have _proper_ base env, we have cookie retention 'cause isolation. Usually this is fine, but for this test we run in JSDOM/browser-like, so we need to do a bit more hygiene to ensure isolation